front/app/components/admin/ContentBuilder/FullscreenPreview/Wrapper.tsx

```diff
@@ -58,7 +60,7 @@ export const FullScreenPreviewWrapper = ({
   return (
     <FocusOn>
       <StyledPreviewBox data-testid="contentBuilderEditModePreviewContent">
-        <Box p="20px">{children}</Box>
+        <Box p={padding || '20px'}>{children}</Box>
```

Comment:
Preview padding is configurable as we need it to be 0px for the homepage.

Reply:
This is used for the mobile and tablet preview where you can directly see the current state of the page builder. We will use the same ContentViewer to display it on the homepage, exactly the same mechanism as the project description has. I just haven't gotten to the display part yet.

Reply:
You can also use `DEFAULT_PADDING` here